cached_credentials.py

```diff
@@ -0,0 +1,41 @@
+import logging
```

Comment:
Why do we have the same file in `examples` folder?

Reply:
Those were added by accident. Removed them.